<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateMetaDataWindow</class>
 <widget class="QDialog" name="UpdateMetaDataWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Meta Data</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>754</width>
        <height>753</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="cacheFields">
         <property name="title">
          <string>Cache Fields</string>
         </property>
         <layout class="QFormLayout" name="cacheFieldsFormLayout"/>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="messageFields">
         <property name="title">
          <string>Message Fields</string>
         </property>
         <layout class="QFormLayout" name="messageFieldsFormLayout"/>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="defaultSortOrder">
         <property name="title">
          <string>Default Sort Order</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="fieldNameLabel">
              <property name="text">
               <string>Field Name</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="fieldNameComboBox"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="sortOrderLabel">
              <property name="text">
               <string>Sort Order</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="sortOrderComboBox">
              <property name="currentText">
               <string>Ascending</string>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <item>
               <property name="text">
                <string>Ascending</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Descending</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="addSortOrderButton">
              <property name="text">
               <string>Add Sort Order</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QTableWidget" name="tableWidget">
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>250</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Field</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Sort Order</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string/>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UpdateMetaDataWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>790</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UpdateMetaDataWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>790</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addSortOrderButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateMetaDataWindow</receiver>
   <slot>addSortOrder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UpdateMetaDataWindow</receiver>
   <slot>saveData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>635</x>
     <y>780</y>
    </hint>
    <hint type="destinationlabel">
     <x>677</x>
     <y>795</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addSortOrder()</slot>
  <slot>saveData()</slot>
 </slots>
</ui>
